recipes/pytest-monitor/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "pytest-monitor" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/CFMTech/pytest-monitor/archive/{{ name }}-{{ version }}.tar.gz
+  sha256: b4081347adcbc13c23eac3384a3dfa83a1c6301bb3aaae90e5588ec30a231ecd
+
+build:
+  number: 0
```

This can be noarch as it doesn't have any native code.
```suggestion
  number: 0
  noarch: python
```

---

Hello

Thanks for your feedback.
I don't support windows and I am not willing to. This is the reason why I put the build selector. The point with this approach is that it forbids me to have noarch packages.

---

Building packages separate for all Python version even if it isn't needed wastes CI resources and introduces a maintenance burden. Feel free to close issues in the package issue tracker about this but I don't think we should put this burden artifically on the conda-forge shoulders.

---

@xhochy I understand your point. I'll see what I can do to make it pass. Thanks for your review.